go/enclave/main/entry.sh

```diff
@@ -11,21 +11,11 @@ if [ ! -L /dev/sgx/enclave ]; then
 	ln -s /dev/sgx_enclave /dev/sgx/enclave
 fi
 
-# If the PCCS_ADDR, the host is provided
-# Do not use the default PCCS_URL defined in /etc/sgx_default_qcnl.conf
-# Particularly used in Alibaba cloud
-if [ -n "${PCCS_ADDR}" ]; then
-	PCCS_URL=https://${PCCS_ADDR}/sgx/certification/v3/
-fi
+PCCS_URL=https://global.acccache.azure.net/sgx/certification/v4/
```

Comment:
This change removes the support that Pedro put in for other cloud providers by hardcoding it to azure, that ok for now?

Reply:
I think it's ok. We'll have to try again anyway